src/main/java/de/muenchen/isi/api/dto/abfrageInBearbeitungSachbearbeitung/BauleitplanverfahrenInBearbeitungSachbearbeitungDto.java

```diff
@@ -25,4 +26,7 @@ public class BauleitplanverfahrenInBearbeitungSachbearbeitungDto extends Abfrage
     private List<
         @NotNull @Valid AbfragevarianteBauleitplanverfahrenInBearbeitungSachbearbeitungDto
     > abfragevariantenSachbearbeitung;
+
+    @Valid
+    private VerortungDto verortung;
```

Comment:
Das Attribut `verortung` ungefähr dort einordnen wo es sich auch in der Klasse `BauleitplanverfahrenDto` befindet.

Also hier vor den beiden List-Attributen.